index.html

```diff
@@ -107,6 +112,17 @@ <h1 class="header">♣️ Basics - Blackjack ♠️</h1>
         // Reset input value
         input.value = "";
       });
+
+      var instruction = document.querySelector("#myBtn");
+      instruction.addEventListener("click", function () {
+
+      var output = document.querySelector("#output-div");   
+      output.innerHTML = howToPlay();
+     });
```

Comment:
Thanks for taking the initiative to explore these DOM manipulations! Good job, can tell you grasp it well.